python/boat_controller_node.py

```diff
@@ -24,8 +24,18 @@
 # Global variables for configuring controller (should be command line arguments ideally)
 DISABLE_LOW_POWER = False
```

Comment:
@patrick-5546 Similar to the other review, I did this for disabling low power mode as well.